core/util/src/test/scala/net/liftweb/util/CanResolveAsyncSpec.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2015 WorldWide Conferencing, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.liftweb
+package util
+
+import scala.concurrent.{Future, Promise}
+
+import org.specs2.mutable.Specification
+
+import actor.LAFuture
+
+object CanResolveAsyncSpec extends Specification {
+  "CanResolveAsync" should {
+    "resolve Scala Futures" in {
+      val myPromise = Promise[String]()
+
+      val resolver = implicitly[CanResolveAsync[Future[String], String]]
```

Comment:
Best way to phrase it is that it fills in what the compiler would give you if you wanted to pass this to an implicit parameter.